src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -105,87 +112,94 @@ Rate limits as enforced by Azure will be shared between any current polling inte
     id="nr-costs"
     title="New Relic costs"
   >
+Standard New Relic billing factors apply. For details, see [New Relic pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing). 
 
-**Standard data ingest costs apply**, based on your New Relic pricing plan. Multiple controls exist to control the amount of data being ingested, including inclusion/exclusion filters and [data dropping rules](/docs/data-apis/manage-data/drop-data-using-nerdgraph/).
   </Collapser>
 </CollapserGroup>
 
-## Setting up the Azure Monitor integration
+## Enable the Azure Monitor integration [#set-up]
+
+Below are instructions for setting up the Azure Monitor integration from New Relic. Alternatively you can use the [Azure Native New Relic Service](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native), which uses this integration. 
+
+To enable the Azure Monitor integration:
+
+1. Follow the instructions for [activating Azure integrations](/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations). 
+2. Go to: **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Azure**, and look for the **Azure Monitor** integration. 
 
-1. See the [Activate Azure integrations](/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations/) documentation. Then the **Azure Monitor** integration will be visible to set up in the Azure integrations tab.
-2. Wait a few minutes for New Relic to begin collecting data for your app.
+Note that it might take a few minutes for data to show up.
 
 <img
```

Comment:
For alt titles, it's worth remembering that they're used by people who have impaired vision or no vision or simply can't load the image. Is this the most useful alt text we can use here?